README.md

```diff
@@ -24,7 +24,7 @@ following these steps:
 * Install [Conan] >= 1.7
 * Add the OSP Conan repository as a remote: 
 
-       conan remote add osp https://osp.jfrog.io/artifactory/api/conan/local
+       conan remote add osp https://osp.jfrog.io/artifactory/api/conan/conan-local
 * Package revisions must be enabled. See [How to activate the revisions].
```

Comment:
This actually also removes the requirement to enable package revisions.

Reply:
So remove the part mentioning package revisions in the README?

Reply:
Yes 👍